charts/incubator/plextraktsync/templates/_cronjob.tpl

```diff
@@ -1,52 +1,30 @@
 {{/* Define the cronjob */}}
 {{- define "plextraktsync.cronjob" -}}
-{{- $jobName := include "tc.common.names.fullname" . }}
-
----
-apiVersion: batch/v1
-kind: CronJob
-metadata:
-  name: {{ printf "%s-cronjob" $jobName }}
-  labels:
-    {{- include "tc.common.labels" . | nindent 4 }}
-spec:
-  schedule: "{{ .Values.plexTraktSync.schedule }}"
-  concurrencyPolicy: Forbid
-  {{- with .Values.cronjob.failedJobsHistoryLimit }}
-  failedJobsHistoryLimit: {{ . }}
-  {{- end }}
-  {{- with .Values.cronjob.successfulJobsHistoryLimit }}
-  successfulJobsHistoryLimit: {{ . }}
-  {{- end }}
-  jobTemplate:
-    metadata:
-    spec:
-      template:
-        metadata:
-        spec:
-          securityContext:
-            runAsUser: 0
-            runAsGroup: 0
-          restartPolicy: Never
-          containers:
-            - name: {{ .Chart.Name }}
-              securityContext:
-                privileged: false
-                readOnlyRootFilesystem: false
-                allowPrivilegeEscalation: false
-                runAsNonRoot: false
-                capabilities:
-                  drop:
-                    - ALL
-              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-              command:
-                - plextraktsync
-                {{- if .Values.plexTraktSync.task }}
-                - {{ .Values.plexTraktSync.task }}
-                {{- else }}
-                - sync --sync=all
-                {{- end }}
-              resources:
-{{ toYaml .Values.resources | indent 16 }}
-
+enabled: true
+type: "CronJob"
+schedule: "{{ .Values.plextraktsync.schedule }}"
+podSpec:
+  restartPolicy: Never
+  containers:
+    cron:
+      enabled: true
+      primary: true
+      imageSelector: "image"
+      command:
+          - /bin/bash
+          - -c
+          - |
+            plextraktsync
+            {{- if  .Values.plextraktsync.task }}
+              {{ .Values.plextraktsync.task }}
+            {{- else }}
+              sync --sync=all
+            {{- end }}
```

Comment:
```suggestion
          - plextraktsync
          {{- if  .Values.plextraktsync.task }}
          - {{ .Values.plextraktsync.task }}
          {{- else }}
          - sync --sync=all
          {{- end }}
```
or this

Reply:
ah i figured i should have kept it like that.